qiskit/circuit/library/standard_gates/dcx.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Double-CNOT gate."""
+
+import numpy as np
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+
+class DCXGate(Gate):
```

Comment:
@ajavadia i really hat this name -- If we cant think of a better im good.

Reply:
agreeing for now this is like the paper from whaley